### PR TITLE
address QE review #1520

### PR DESCRIPTION
**Pull request by domiborges**
Follows up on #1520 

Address QE review:
Include additional info about ADDITIONAL_VARS
Add step to recreate the cluster

### CODE REVIEW

File: documentation/asciidoc/topics/proc_defining_environment_var.adoc [hunk 1]
```diff
@@ -9,8 +9,10 @@ ifdef::downstream[]
 NOTE: If you are using the {openshift} Console, you must first install the {ispn_operator} and then edit the existing subscription.
 endif::downstream[]
 
-Environment variables in your subscription automatically propagate to each {brandname} pod that is managed by the corresponding Operator.
-The `spec.config.env` field is an array of environment variable definitions.
+Define environment variables using the `spec.config.env` field.
+Set the name of the first variable as `ADDITIONAL_VARS`.
+This variable is a JSON array that includes the names of environment variables to be automatically propagated to each {brandname} Server pod managed by the associated Operator.
+
 When one environment variable references another, you must define the variables in a specific order for proper referencing and processing.
```
Comment (Crumby) [inline documentation/asciidoc/topics/proc_defining_environment_var.adoc:16]: Do we need this sentence here given it's general rule for any OpenShift resource (cc @ryanemerson )? It's not related to propagating env vars to the server pods

File: documentation/asciidoc/topics/proc_defining_environment_var.adoc [hunk 2]
```diff
@@ -9,8 +9,10 @@ ifdef::downstream[]
 NOTE: If you are using the {openshift} Console, you must first install the {ispn_operator} and then edit the existing subscription.
 endif::downstream[]
 
-Environment variables in your subscription automatically propagate to each {brandname} pod that is managed by the corresponding Operator.
-The `spec.config.env` field is an array of environment variable definitions.
+Define environment variables using the `spec.config.env` field.
+Set the name of the first variable as `ADDITIONAL_VARS`.
```
Comment (Crumby) [inline documentation/asciidoc/topics/proc_defining_environment_var.adoc:13]: Technically does not need to be first

File: documentation/asciidoc/topics/proc_defining_environment_var.adoc (hunk 1 above)
Comment (ryanemerson) [reply documentation/asciidoc/topics/proc_defining_environment_var.adoc:16]: That's true. Let's keep it focused and remove this.

File: documentation/asciidoc/topics/proc_defining_environment_var.adoc (hunk 2 above)
Comment (domiborges) [reply documentation/asciidoc/topics/proc_defining_environment_var.adoc]: Updated as well

**Review by Crumby — APPROVED** [review]

**Pull request merged**